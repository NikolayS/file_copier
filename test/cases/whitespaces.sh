#!/bin/bash

# to run this scenario, COPIERSERVICE should be defined (`export COPIERSERVICE=https://your.copier.host.name`)

http GET "$COPIERSERVICE/?wh=1&md5=1&src=https://3.bp.blogspot.com/-eICF8Px0Eyc/WBBnc6vI51I/AAAAAAAANac/pxfFy2YvTfcl09MgwQ_aHO-ZHHR7XakJgCLcB/s640/Doar%20o%20lingurita%20din%20aceasta%20apa%20de%20gura%20facuta%20in%20casa%20te%20scapa%20de%20halena,%20carii%20si%20paradontoza.....jpg" -h 

