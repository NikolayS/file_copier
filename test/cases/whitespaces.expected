HTTP/1.1 200 OK
X-File-Copier-Md5: c2bb5a87fd3a2e6ff5e7ab43b570854e
X-File-Copier-Img-Width: 600
X-File-Copier-Img-Height: 320
X-File-Copier-Size: 65170
X-Location: /data/1c/87/86/f1/1c8786f14cf968228456743319cba6812aca0835168c5997b753e6f40f7b4473.jpg
Access-Control-Expose-Headers: X-File-Copier-Size, X-Location, X-File-Copier-Img-Height, X-File-Copier-Img-Width, X-File-Copier-Md5

